sdk/containerregistry/Microsoft.Azure.ContainerRegistry/src/Microsoft.Azure.ContainerRegistry.csproj

```diff
@@ -3,7 +3,7 @@
   	<PackageId>Microsoft.Azure.ContainerRegistry</PackageId>
     <Description>This is a client library for Azure Container Registry data plane operations. Use this library to retrieve, update and view Registry tags, blobs and manifests. </Description>
     <AssemblyTitle>Microsoft Azure Container Registry</AssemblyTitle>
-    <Version>0.10.1-preview</Version>
+    <Version>0.10.1-preview.1</Version>
```

Comment:
Is there a reason for starting the major from 0?
cc @weshaggard

Reply:
Should I bump to 1.0.0-preview.1... to make things go quickly, hoping to release to unblock  the team that raised the issue

Reply:
That will be ideal except your team has specific to do otherwise

Reply:
thanks @chidozieononiwu, done